### ISSUE TITLE
Accessibility! Tab navigation and color contrast issues

### ISSUE THREAD

**Issue by sodevious** [DEFNA/defna.org#15]
## Nav dropdowns should be keyboard navigable

- should open/toggle with either enter, space or down key
- stay open until the user closes or leaves the dropdown

##  Donate button fails WCAG contrast ratio (1.68:1). Must be 3:1

<img width="1206" height="287" alt="Image" src="https://github.com/user-attachments/assets/33abc2d4-c800-4b6b-b1b7-ceb3c0656405" />

## Footer text color fails WCAG contrast ratio for small text

<img width="1218" height="281" alt="Image" src="https://github.com/user-attachments/assets/cf947c6e-ad75-42f1-aba1-48d040dc8749" />

## Yellow-on-tan very hard to see, also some labels fail WCAG

<img width="628" height="130" alt="Image" src="https://github.com/user-attachments/assets/636cd29b-f2e6-41ab-beb6-f8e9c2afdd75" />
